docs/demos/basics/index.txt

```diff
@@ -0,0 +1,9 @@
+basics
+====================================
+
+.. toctree::
```

Comment:
Is there a way to capitalise the contents headings to match the rest of the docs? (possibly just re-name the folders, unless there's a sphinx way)